app/src/main/java/cm/aptoide/pt/app/view/donations/utils/GenericPaymentIntentBuilder.java

```diff
@@ -43,19 +41,7 @@
   public static PendingIntent buildBuyIntent(Context context, String skuId, String value,
       String walletAddress, String packageName, String transferType, String payload,
       boolean debug) {
-    AppCoinsAddressProxySdk proxySdk = new AppCoinsAddressProxyBuilder().createAddressProxySdk();
-    int networkId = debug ? ROPSTEN_NETWORK_ID : MAIN_NETWORK_ID;
-
-    Single<String> getTokenContractAddress = proxySdk.getAppCoinsAddress(networkId)
-        .subscribeOn(Schedulers.io());
-    Single<String> getIabContractAddress = proxySdk.getIabAddress(networkId)
-        .subscribeOn(Schedulers.io());
-
-    return Single.zip(getTokenContractAddress, getIabContractAddress,
-        (tokenContractAddress, iabContractAddress) -> buildPaymentIntent(context, networkId, skuId,
-            value, tokenContractAddress, iabContractAddress, walletAddress, packageName,
-            transferType, payload))
-        .blockingGet();
+    return null;
```

Comment:
maybe safer to return Single.just() ?

Reply:
Sure.

Reply:
Actually, it doesn't build with that line.